app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/conversationList.tsx

```
@@ -79,14 +91,20 @@ export const List = () => {
-      const conversationFilter = allConversationsSelected
-        ? conversations.length <= 25 && !hasNextPage
-          ? conversations.map((c) => c.id)
-          : input
-        : selectedConversations;
+      const conversationFilter = allConversationsSelected ? input : selectedConversations;
+      let selectedCount = allConversationsSelected ? conversations.length : selectedConversations.length;
+
+      if (allConversationsSelected) {
+        try {
+          const countResult = await utils.mailbox.conversations.count.fetch(input);
+          selectedCount = countResult.total;
```

ESLint

[error] 'selectedCount' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)